modules/services/mapillary.js

```diff
@@ -245,43 +268,37 @@ export default {
 
     signHTML: function(d) {
         if (!mapillarySignDefs) return;
-
-        var detectionPackage = d.signs[0].package,
-            type = d.signs[0].type,
-            country = detectionPackage.split('_')[1];
-
-        return mapillarySignDefs[country][type];
+        var position = mapillarySignDefs[d.value];
+        if (!position) return '<div></div>';
+        var iconStyle = [
+            'background-image:url(img/traffic-signs/traffic-signs.png)',
```

Comment:
I think I need to run this through `context.asset()` because once this is on the Rails site, the asset pipeline may rewrite the URL.